systemdBoot.js

```diff
@@ -36,8 +36,15 @@ const Utils = Me.imports.utils;
  * they will only show up on the next boot.
  */
 async function getBootOptions() {
+    let bootctl = Utils.getBootctlPath();
+    if (bootctl == "") {
+        Utils._log(`Failed to find bootctl binary`);
+        return undefined;
+    }
+    Utils._log(`bootcl found at ${bootctl}`);
```

Comment:
I'm not sure if this needs to be logged.

Reply:
I forgot to remove that, was only for testing purposes